File: java/url/src/main/java/com/cleutonsampaio/dojo/url/UrlApplication.java
package com.cleutonsampaio.dojo.url;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.autoconfigure.jdbc.DataSourceAutoConfiguration;

@SpringBootApplication(exclude = { DataSourceAutoConfiguration.class})
public class UrlApplication {

	public static void main(String[] args) {
		SpringApplication.run(UrlApplication.class, args);
	}

}

File: kotlin/url/src/main/kotlin/com/cleutonsampaio/dojo/url/UrlApplication.kt
package com.cleutonsampaio.dojo.url

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.autoconfigure.jdbc.DataSourceAutoConfiguration
import org.springframework.boot.runApplication

@SpringBootApplication(exclude = arrayOf(DataSourceAutoConfiguration::class))
class UrlApplication

fun main(args: Array<String>) {
	runApplication<UrlApplication>(*args)
}
